.travis.yml

```diff
@@ -119,14 +120,38 @@ before_install:
       alias make="${SCAN_BUILD} make"
     fi
 
+  ############################################################################
+  # Install brew llvm
+  ############################################################################
+  - |
+    if [[ "${USE_BREW_LLVM}" == "ON" ]]; then
+        brew install llvm;
+        export OMP_NUM_THREADS=4;
+        export LDFLAGS="-L/usr/local/opt/llvm/lib -Wl,-rpath,/usr/local/opt/llvm/lib";
+        export DYLD_LIBRARY_PATH="/usr/local/opt/llvm/lib:$DYLD_LIBRARY_PATH";
+        export CC="/usr/local/opt/llvm/bin/clang";
+        export CXX="/usr/local/opt/llvm/bin/clang++";
+        export CPPFLAGS="-I/usr/local/opt/llvm/include $CPPFLAGS";
+        export CFLAGS="-I/usr/local/opt/llvm/include $CFLAGS";
+    fi
+
+  ############################################################################
+  # Install OpenBLAS
+  ############################################################################
+  - |
+    if [[ "${TRAVIS_OS_NAME}" == "osx" && "${BACKEND}" == "OpenBLAS" ]]; then
+        brew install openblas;
+        brew link openblas --force;
+    fi
+
 script:
   - cd ${TRAVIS_BUILD_DIR}
   - mkdir -p build && cd build
   - |
     if [[ $BUILD_PACKAGE == "ON" && $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_BRANCH == "master" ]]; then
-      export CMAKE_OPTIONS="-DCMAKE_CXX_STANDARD=11 -DBLA_VENDOR=$BACKEND"
+      export CMAKE_OPTIONS=" -DBLA_VENDOR=$BACKEND"
     else
-      export CMAKE_OPTIONS="-DCMAKE_CXX_STANDARD=11 -DBLA_VENDOR=$BACKEND -DENABLE_COVERAGE=ON"
+      export CMAKE_OPTIONS=" -DBLA_VENDOR=$BACKEND -DENABLE_COVERAGE=ON"
```

Comment:
@massich I remove this as set it in the cmake. no need to add this to cmkae line all the time.

Reply:
I disagree, I would rather take it out from cmake. But I can do it in a PR when adding specific compilation flags.

Reply:
pb is that cmake then does not work out of the box and you need cleverness to be able to build...

Reply:
ideally doing:

mkdir build
cd build
cmake .
make

should work provided you have installed the dependencies on your sytem with brew, conda or apt-get depending on your system. Cf. remark from @papadop about having good defaults for BLA_VENDOR